ci/packer/zen/Makefile

```diff
@@ -37,15 +41,15 @@ zen-release:
 	make \
 	PKR_VAR_pf_repo=packetfence \
 	PKR_VAR_pf_package=packetfence \
-	PKR_VAR_pf_branch=stable \
+	PKR_VAR_pf_release_pkg=packetfence-release-$(PF_MINOR_RELEASE)-el7.noarch.rpm \
```

Comment:
What will happened for rhel8?

Reply:
Because we will only support one EL group at a time, I will replace el7 by el8 on `feature/rhel8` branch.